.github/workflows/preDeploy.yml

```diff
@@ -337,7 +337,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Check if test failed, if so leave a deploy blocker label
-        if: ${{ !contains(env.OUTPUT, 'There are no entries') }}
+        if: ${{ contains(env.OUTPUT, 'Significant Changes To Duration') && contains(env.OUTPUT, '🔴') }}
```

Comment:
I'm pretty sure `Significant Changes To Duration` will always show up since it's the header. I don't think we need the first check. All we need to do is check for the 🔴 
```suggestion
        if: ${{ contains(env.OUTPUT, '🔴') }}
```

Reply:
Example: https://github.com/Expensify/App/pull/13703#issuecomment-1358416787